mittab/libs/tab_logic.py

```diff
@@ -619,13 +638,13 @@ def avg_deb_speaks(debater):
         return float(sum(real_speaks)) / float(len(real_speaks))
 
 
-def debater_forfeit_speaks(debater):
-    """ Calculate a debater's speaks for a forfeit round
+def debater_forfeit_speaks(debater, round_number):
+    #Calculate a debater's speaks for a forfeit round
 
-    Note that right now we just return 0, but we may want to add support
-    for returning average speaks or some such
-    """
-    return 0.0
+    if (TabSettings.get('lenient_late') >= round_number):
```

Comment:
So the reason for this implementation is twofold. First, W&M's tab policy works this way (if you call before Round 1 is posted to tell us you'll be late, we drop you with avg speaks, as our tournament is so far south travel issues are common), I wasn't sure how other schools manage their late arrivals because I've never been affected by it at other tournaments. 

Secondly, this version of the implementation was much more easier to implement than a case-by-case basis, and I'm lazy. Doing case-by-case checks would require either substantially modifying the bye table or adding a separate table. The followup issue is that mit-tab can't predict when tab directors will justify noshows, so you either need individual prompts for each checked out team in every round, which seems excessive in the event that a team quits mid-tournament, or the tab director has to go into admin to manually add a team as being a lenient noshow. If there's a way to do case-by-case checks without tons of prompts, or you think having the prompts is fine, I'd be happy to re-do it.